index.py

```diff
@@ -6,17 +6,19 @@
 
 app = Flask(__name__)
 
-_kbase_url = os.environ.get('KBASE_ENDPOINT', 'https://ci.kbase.us/services')
 
+
+app.config['APPLICATION_ROOT'] = os.environ.get('ROOT_PREFIX', '/catalog')
+# app.url_map._rules = SubPath(app.config['APPLICATION_ROOT'], app.url_map._rules)
+# print('*' * 80)
+# print(app.config['APPLICATION_ROOT'])
+
+_kbase_url = os.environ.get('KBASE_ENDPOINT', 'https://ci.kbase.us/services')
```

Comment:
I feel like it's probably best to just fail here if the KBASE_ENDPOINT doesn't exist. A user might get confused if this is misconfigured in production and shows the CI Apps, instead of just failing to start.